### DISCUSSION TITLE
Renovate not able to fetch docker dependencies from Private Registry

### DISCUSSION THREAD

**Discussion by rajat053117** [renovatebot/renovate#28463] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Github

### Please tell us more about your question or problem

Hi Team , 
I am using a repo A which fetches the dependencies for  Package this ---> us-east1-docker.pkg.dev/d-gs-secops-cp/falcon/falcon-sensor:7.11.0-16404-1.falcon-linux.x86_64.Release.US-2 and updates the same in my private registry and i can see the new dependency update. 

Now i am using another repo B which checks this package from the artifact registry and updates the dependency in my another repo but the issues is it is not able to update the  same package in another repo 

### Logs (if relevant)

<details><summary>Logs</summary>
Renovate.json is as below 

```
    {
      customType: 'regex',
      fileMatch: [
        'abc.gotmpl$'
      ],
      matchStrings: [
        'repository:\\s+(?<depName>[^:\\n]+)\\s*\\n\\s+tag:\\s+"?(?<currentValue>[^@\\n]+)@(?<currentDigest>sha256:[a-f0-9]+)"?',
      ],
      datasourceTemplate: 'docker',
      versioningTemplate: 'docker',
    },
  ],
and a package rule as 
    {
      matchDatasources: [
        'docker',
      ],
      matchPackageNames: [
        'us-east1-docker.pkg.dev/d-gs-secops-cp/falcon/falcon-sensor',         //Ensure falcon-senor use loose versioning for non-semver tags
      ],
      versioning: 'loose',
    }
    
contents of abc.gotmpl looks like 
    repository: xxxxxxx
    tag: 7.03.0-15805.falcon-linux.x86_64.Release.US-2@sha256:xxxxxxxxx    
```
I think what i am doing wrong is using  versioningTemplate: 'docker', Instead use something like 'loose' 
Not sure if anyone can help me on this , i would be thankful

**Comment by rarkins** [comment, marked as answer]
Yes try loose versioning
